Cargo.toml

```diff
@@ -33,6 +33,7 @@ tls-vendored = ["native-tls/vendored"]
 clap = "2.33.3"
 ansi_term = "0.12.1"
 dirs-next = "2.0.0"
+directories-next = "2.0.0"
```

Comment:
It seems to me like `directories-next` is a superset of `dirs-next`.
Since `directories-next` also has a `home_dir()` method, we can use the opportunity to replace `dirs-next` calls altogether and remove the dependency.

Reply:
Done!